h/search/index.py

```diff
@@ -60,7 +60,7 @@ def index(es, annotation, request, target_index=None):
     )
 
 
-def delete(es, annotation_id, target_index=None):
+def delete(es, annotation_id, target_index=None, refresh=False):
```

Comment:
We do want the deletion to be immediately visible, don't we? Shouldn't this default to True?

Reply:
In tests we do want the deletion to be immediately visible. In the app the current behaviour is that `refresh` is `False` which means that the change becomes visible after a short delay (of about a second or so). I've left the default as `False` here to avoid changing behaviour in the actual app.

I personally think we might want to change the behaviour of the API in production at some point to make it so that delete operations are guaranteed to be immediately in search responses, but that's a separate exercise.

Reply:
Sounds good.